scimath/interpolate/fitting.py

```diff
@@ -82,21 +85,26 @@ def block_interp(self, x):
         # This code is a little strange -- we really want a routine that
         # returns the index of values where x[j] < x[index]
         TINY = 1e-10
-        indices = numpy.searchsorted(self._x, x+TINY)-1
+        indices = numpy.searchsorted(self._x, x + TINY) - 1
 
         # If the value is at the front of the list, it'll have -1.
         # In this case, we will use the first (0), element in the array.
         # take requires the index array to be an Int
-        indices = numpy.atleast_1d(numpy.clip(indices, 0, numpy.Inf).astype(numpy.int))
+        indices = numpy.atleast_1d(
+            numpy.clip(
+                indices,
+                0,
+                numpy.Inf).astype(
+                numpy.int))
```

Comment:
Nit: stylistically, I'd break this up into two lines with a temporary to fit in the linewidth constraints.

```
        indices = numpy.clip(indices, 0, numpy.Inf).astype(numpy.int)
        indices = np.atleast_1d(indices)
```